src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Undo previous change to AddresSoc.
```

Comment:
Should this be phrased differently to show that it's different from UndoCommand? maybe `Redo the last action that was undone` or something along those lines